sandbox/templates/richie/section/_highlighted_items.scss

```diff
@@ -0,0 +1,94 @@
+$richie-section-highlights-padding: 3rem 1rem !default;
+$richie-section-highlights-background-even: $white !default;
+$richie-section-highlights-background-odd: $gray98 !default;
+$richie-section-highlights-title-color: $gray17 !default;
+$richie-section-highlights-title-margin: 0 auto 1rem !default;
+$richie-section-highlights-title-padding: 1rem !default;
+$richie-section-highlights-title-textalign: center !default;
+$richie-section-highlights-item-gutter: 0.625rem !default;
+
+// TODO: unrelated: Better Hero on large_banner template "hero-intro.html"
+.section{
+    $base-section-selector: &;
+
+    &--highlights{
+        padding: $richie-section-highlights-padding;
+        background: $richie-section-highlights-background-even;
+
+        &:nth-child(odd){
+            background: $richie-section-highlights-background-odd;
+        }
+
+        #{$base-section-selector}__title {
+            margin: $richie-section-highlights-title-margin;
+            padding: $richie-section-highlights-title-padding;
+            color: $richie-section-highlights-title-color;
+            text-align: $richie-section-highlights-title-textalign;
+            @include media-breakpoint-up(md) {
+                @include make-container();
+                @include make-container-max-widths();
+            }
+        }
+
+        #{$base-section-selector}__items {
+            display: flex;
+            flex-direction: row;
```

Comment:
is `row` not the default direction?

Reply:
Yes it is, but personally i like to be explicit because some old browser didn't always have the right defaults and leads sometime to some bugs, like for explicitly defines flex grow, shrink and basis instead of the shortcut `flex`.

Maybe we should remove it if compatibility is not an issue with some browser from ~3years ago